File: 프로그래머스/unrated/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.cs
using System;

public class Solution {
    public int[] solution(int[] arr, int[,] queries) 
    {
        for(int y = 0; y < queries.GetLength(0); ++y)
        {
            int i = queries[y, 0];
            int j = queries[y, 1];
            
            int temp = arr[i];
            arr[i] = arr[j];
            arr[j] = temp;
        }
        return arr;
    }
}

File: 프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    
    for (const vector<int> v : queries)
        swap(arr[v[0]], arr[v[1]]);
    
    return arr;
}